tests/spec/core/linter-rules/check-charset-spec.js

```diff
@@ -0,0 +1,72 @@
+describe("Core Linter Rule - 'check-charset'", () => {
+  const ruleName = "check-charset";
+  const config = {
+    lint: { [ruleName]: true },
+  };
+  let rule;
+  beforeAll(async () => {
+    rule = await new Promise(resolve => {
+      require([`core/linter-rules/${ruleName}`], ({ rule }) => resolve(rule));
+    });
+  });
+
+  it("checks if meta[charset] is set to utf-8", async () => {
+    const doc = document.implementation.createHTMLDocument("test doc");
+    doc.head.innerHTML = `
+      <meta charset="utf-8">
+      <meta name="viewport" content="width=device-width">
+      <title>ReSpec</title>
+    `;
+
+    const results = await rule.lint(config, doc);
+    expect(results.length).toEqual(0);
+  });
+
+  it("doesn't give an error when written in capitals", async () => {
+    const doc = document.implementation.createHTMLDocument("test doc");
+    doc.head.innerHTML = `
+      <meta charset="UTF-8">
+      <meta name="viewport" content="width=device-width">
+      <title>ReSpec</title>
+    `;
+
+    const results = await rule.lint(config, doc);
+    expect(results.length).toEqual(0);
+  });
+
+  it("checks if meta[charset] is present or not", async () => {
+    const doc = document.implementation.createHTMLDocument("test doc");
+    doc.head.innerHTML = `
+      <meta name="viewport" content="width=device-width">
+      <title>ReSpec</title>
+    `;
+
+    const results = await rule.lint(config, doc);
+    expect(results.occurrences).toEqual(0);
+  });
+
+  it("returns error when more then one meta[charset] present", async () => {
+    const doc = document.implementation.createHTMLDocument("test doc");
+    doc.head.innerHTML = `
+      <meta charset="utf-8">
+      <meta charset="ascii-128">
+      <meta name="viewport" content="width=device-width">
+      <title>ReSpec</title>
+    `;
+
+    const results = await rule.lint(config, doc);
+    expect(results.occurrences).toEqual(2);
```

Comment:
I feel this should be 1, @marcoscaceres Thoughts?

Reply:
I'm sending `metas.length` so I guess it'll be 2, if I send `val.length` it'll be 1

Reply:
Hmm... okay, 2 is good. Never mind 😁